lisk-dex-electron/src/config/Configuration.js

```diff
@@ -69,10 +70,9 @@ const DEXConfiguration = Record({
   })),
 });
 
-export default async function processConfiguration(config) {
+export default async function createRefinedGlobalConfig(config) {
```

Comment:
Maybe we can rename this to `extendGlobalConfig` or similar? It's mutating the existing `GlobalConfiguration` object so it would be nice if the function name could indicate that this mutation is occurring.

Reply:
I think it's better if we don't mutate stuff in createRefinedGlobalConfig, we just return data and then initialize config